packages/design-tokens/css/heart-theme.css

```diff
@@ -1,301 +1,1694 @@
+  --kz-var-border-borderless-border-color-rgb: 0, 0, 0;
+  --kz-var-border-borderless-border-color-rgb-id: --kz-var-border-borderless-border-color-rgb;
   --kz-var-border-borderless-border-color-rgb-params: 0, 0, 0;
+  --kz-var-border-borderless-border-color-id: --kz-var-border-borderless-border-color;
   --kz-var-border-focus-ring-border-width: 2px;
+  --kz-var-border-focus-ring-border-width-id: --kz-var-border-focus-ring-border-width;
   --kz-var-border-focus-ring-border-radius: 10px;
+  --kz-var-border-focus-ring-border-radius-id: --kz-var-border-focus-ring-border-radius;
   --kz-var-border-focus-ring-border-style: solid;
+  --kz-var-border-focus-ring-border-style-id: --kz-var-border-focus-ring-border-style;
   --kz-var-color-wisteria-100: #f4edf8;
+  --kz-var-color-wisteria-100-rgb: 244, 237, 248;
+  --kz-var-color-wisteria-100-rgb-id: --kz-var-color-wisteria-100-rgb;
   --kz-var-color-wisteria-100-rgb-params: 244, 237, 248;
+  --kz-var-color-wisteria-100-id: --kz-var-color-wisteria-100;
   --kz-var-color-wisteria-200: #dfc9ea;
+  --kz-var-color-wisteria-200-rgb: 223, 201, 234;
+  --kz-var-color-wisteria-200-rgb-id: --kz-var-color-wisteria-200-rgb;
   --kz-var-color-wisteria-200-rgb-params: 223, 201, 234;
+  --kz-var-color-wisteria-200-id: --kz-var-color-wisteria-200;
   --kz-var-color-wisteria-300: #c9a5dd;
+  --kz-var-color-wisteria-300-rgb: 201, 165, 221;
+  --kz-var-color-wisteria-300-rgb-id: --kz-var-color-wisteria-300-rgb;
   --kz-var-color-wisteria-300-rgb-params: 201, 165, 221;
+  --kz-var-color-wisteria-300-id: --kz-var-color-wisteria-300;
   --kz-var-color-wisteria-400: #ae67b1;
+  --kz-var-color-wisteria-400-rgb: 174, 103, 177;
+  --kz-var-color-wisteria-400-rgb-id: --kz-var-color-wisteria-400-rgb;
   --kz-var-color-wisteria-400-rgb-params: 174, 103, 177;
+  --kz-var-color-wisteria-400-id: --kz-var-color-wisteria-400;
   --kz-var-color-wisteria-500: #844587;
+  --kz-var-color-wisteria-500-rgb: 132, 69, 135;
+  --kz-var-color-wisteria-500-rgb-id: --kz-var-color-wisteria-500-rgb;
   --kz-var-color-wisteria-500-rgb-params: 132, 69, 135;
+  --kz-var-color-wisteria-500-id: --kz-var-color-wisteria-500;
   --kz-var-color-wisteria-600: #5f3361;
+  --kz-var-color-wisteria-600-rgb: 95, 51, 97;
+  --kz-var-color-wisteria-600-rgb-id: --kz-var-color-wisteria-600-rgb;
   --kz-var-color-wisteria-600-rgb-params: 95, 51, 97;
+  --kz-var-color-wisteria-600-id: --kz-var-color-wisteria-600;
   --kz-var-color-wisteria-700: #4a234d;
+  --kz-var-color-wisteria-700-rgb: 74, 35, 77;
+  --kz-var-color-wisteria-700-rgb-id: --kz-var-color-wisteria-700-rgb;
   --kz-var-color-wisteria-700-rgb-params: 74, 35, 77;
+  --kz-var-color-wisteria-700-id: --kz-var-color-wisteria-700;
   --kz-var-color-wisteria-800: #2f2438;
+  --kz-var-color-wisteria-800-rgb: 47, 36, 56;
+  --kz-var-color-wisteria-800-rgb-id: --kz-var-color-wisteria-800-rgb;
   --kz-var-color-wisteria-800-rgb-params: 47, 36, 56;
+  --kz-var-color-wisteria-800-id: --kz-var-color-wisteria-800;
```

Comment:
Same for this file (and I'm guessing zen-theme.css will have a similar issue). I don't think we should be using the new colour names with the kz prefix (even in our intermittent state)